Development/nmos/system_resources.cpp

```diff
@@ -35,13 +69,32 @@ namespace nmos
         const auto& is04 = nmos::fields::is04(data);
         const auto& ptp = nmos::fields::ptp(data);
 
+        auto global_data = value_of({
+            { nmos::fields::registration_heartbeat_interval, nmos::fields::heartbeat_interval(is04) },
+            { nmos::fields::ptp_announce_receipt_timeout, nmos::fields::announce_receipt_timeout(ptp) },
+            { nmos::fields::ptp_domain_number, nmos::fields::domain_number(ptp) },
+            { nmos::fields::tags, data.at(nmos::fields::tags) }
```

Comment:
This function is returning an `nmos::settings` so it should have `system_tags` not `tags`, right?

Reply:
fixed